src/app/components/content/IsaacSymbolicChemistryQuestion.tsx

```diff
@@ -64,7 +235,43 @@ const IsaacSymbolicChemistryQuestion = ({doc, questionId, readonly}: IsaacQuesti
                 editorMode="chemistry"
                 questionDoc={doc}
             />}
+            {!readonly && <div className="eqn-editor-input">
+                <div ref={hiddenEditorRef} className="equation-editor-text-entry" style={{height: 0, overflow: "hidden", visibility: "hidden"}} />
+                <InputGroup className="my-2 separate-input-group">
+                    <Input type="text" onChange={updateEquation} value={textInput}
+                              placeholder="Type your formula here"/>
+                    <>
+                        {siteSpecific(
+                            <Button type="button" className="eqn-editor-help" id={helpTooltipId} tag="a" href="/solving_problems#symbolic_text">?</Button>,
+                            <span id={helpTooltipId} className="icon-help-q my-auto"/>
+                        )}
+                        {doc.isNuclear
+                                ? <UncontrolledTooltip placement="top" autohide={false} target={helpTooltipId}>
```

Comment:
Nice examples! However I think that with the length of Nuclear answers, the overflow of an example over multiple lines makes it quite unclear when one ends and the next starts. I've got a couple different solutions in mind:

Either we can change each `<br />` to `<br className="mb-2" />` 

OR for the same effect with a more forward-thinking approach add a new className like `spacedTooltip` to `UncontrolledTooltip` 

```suggestion
                                ? <UncontrolledTooltip className="spacedTooltip" placement="top" autohide={false} target={helpTooltipId}>
```
and then in a css file (`questions.scss` should do, at the top-level due to how tooltips are handled): 
``` 
.spacedTooltip br {
  margin-bottom: 0.5rem;
}
```

Reply:
The Chemistry tooltip below, and probably even the Symbolic Maths [tooltip](https://github.com/isaacphysics/isaac-react-app/blob/98cbab5920b7976f7021c3470121d9cbc716ce73/src/app/components/content/IsaacSymbolicQuestion.tsx#L243C1-L243C25) could probably also do with this same `spacedTooltip` treatment, as they also have examples going over multiple lines - although just 2 lines instead of the 3 here.

(I'd also then remove the extra floating `<br />`s either side of the examples - otherwise it's taking up too much space)

Reply:
I think the maths tooltip is okay for now. If people get annoyed at the inconsistency then we can change it.

When this is merged I will also update the symbol help accordion `/solving_problems#symbolic_text` so that there is a more in depth symbol list for students.